### DISCUSSION TITLE
Set brightness curve for each display?

### DISCUSSION THREAD

**Discussion by philippwulff** [MonitorControl/MonitorControl#803] [Ideas]
Hey everyone, 

first of all, I really like this app. I used it a few year ago and just installed it again and is has improved a lot!

When I sync the brightness across my displays everything works, but my issue is that a brightness level of 20% on my external monitor is brighter than a brightness of 20% on my built-in MacBook Screen. Therefore, I would like to set a curve that matches the brightness of the built-in screen to a customised brightness for the external monitor. I could not find any issues/discussions on this or anything in the settings, so I am wondering, if this feature exists. Does it? 

Basically I would love to set a brightness curve for each display like this:
![untitled-2](https://user-images.githubusercontent.com/45896065/142633069-40676f9f-251d-46b2-8214-7bbf752ff2ee.png)

What do you think?

**Comment by philippwulff** [comment]
I just found an [issue](https://github.com/MonitorControl/MonitorControl/issues/570), which basically led to the implementation of this idea. 

I set the slider in scale mapping curve all the way to the right and this seems to achieve the desired output (the curve I drew). But the settings are kind of difficult to understand here. Maybe they should be augmented to show the curve they produce :)

But other than that this solves my problem

**Reply by waydabber** [reply]
Hi, yes, this can be adjusted by changing the "Scale mapping curve" which will affect the DDC controlled portion of the dimming (if combined dimming is enabled, the range above the dimming switchover point). You can play both with the switchover point and the curve + the min/max DDC values to find the best setting for each display and match all displays together.
